### ISSUE TITLE
[Feature Request] Application Blur | Option to ignore full screen windows

### ISSUE THREAD

**Issue by ItsLynix** [aunetx/blur-my-shell#509]
I'd like an option to if application blur is enabled on e.g all apps by default, ignore apps that are full screen, for example games etc.

**Comment by utsav0904** [comment]
+1 to that

**Comment by aunetx** [comment]
Hello, I implemented a similar option in latest commits, that is ignoring focused windows (fullscreen or not). This way, the window you are working on (be it a game, IDE, firefox...) is not blurred so that you have good performances and legibility, and the background windows are blurred.

Is this okay for you?

**Comment by utsav0904** [comment]
This sounds okay but from some focused windows I need blurred screen, e.g., terminal or code editor where I want background blur. By this commit I'll lose blur effect on those use cases so technically it's like not using blur-my-shell.

**Comment by aunetx** [comment]
Of course, this behaviour can be disabled in preferences. I plan to add per-window configuration about it, however that's quite hard to implement! 

I will see about ignoring fullscreen windows too!

**Comment by TiagoCraft** [comment]
thx Aurelien. got to your extension specifically looking for this behaviour!
Slightly bluring BG applicatons drives attention to the focused window and makes more obvious which one has keyboard focus. 
it kind of mimics atmospheric perspective, something i even [tried to do and gave up](https://github.com/TiagoCraft/atmosphere) as i lacked GJS skills to do.

**Comment by voidcityy** [comment]
How is the progress on ignoring fullscreen windows? I wouldn't want to use "opaque focused window" option as I still need the blur effect in CLI, but somehow BMS still adds transparency to some Proton games when I disable that option. Adding them as extensions simply doesn't work (tried both process and window names).

**Comment by MeguMario** [comment]
https://github.com/aunetx/blur-my-shell/pull/965
